llarp/service/endpoint.cpp

```diff
@@ -1054,26 +1057,27 @@ namespace llarp
     {
       if (not NameIsValid(name))
       {
-        handler(ParseAddress(name));
+        log::warning(logcat, "\"{}\" is not a valid ONS name", name);
```

Comment:
feel free to use single quotes in the error message:

` log::warning(logcat, '{}' is not a valid ONS name", name); `